packages/grafana-eslint-rules/rules/no-border-radius-literal.cjs

```diff
@@ -0,0 +1,48 @@
+// @ts-check
+const { ESLintUtils, AST_NODE_TYPES } = require('@typescript-eslint/utils');
+
+const createRule = ESLintUtils.RuleCreator((name) => `https://github.com/grafana/grafana#${name}`);
+
+const borderRadiusRule = createRule({
+  create(context) {
+    return {
+      CallExpression(node) {
+        if (
+          node.callee.type === AST_NODE_TYPES.Identifier &&
+          node.callee.name === 'css' &&
+          node.arguments[0].type === 'ObjectExpression'
```

Comment:
`css()` can also be called with an array of objects e.g. `css([ {width: 10, height: 10} ])`.

You might want to make a utility function that takes each argument node and returns any ObjectExpressions they may contain. 

e.g something like (not tested)

```ts
function getCSSObjects(node: WhateverTheTypeOfArgumentsIs): ObjectExpressionNode[] {
  if (node.type === "ObjectExpression") {
    return [node];
  }

  if (node.type === "ArrayExpression") {
    const children: ObjectExpression[] = [];

    for (const childNode of node.elements) {
      if (childNode.type === "ObjectExpression") {
        children.push(childNode)
      }
    }

    return children;
  }

  return [];
}

// ...

if (node.callee.type === "Identifier" && node.callee.name === "css") {
  const cssObjects = node.arguments.flatMap(argument => getCSSObjects(argument));

  for (const cssObjectNode of cssObjects) {
    for (const property of cssObjectNode.properties) {
      // ...
    
```